tools/check/check-lint.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 # check for io/ioutil
-GREP_IOUTIL=`find -name "*.go" | xargs grep -ns "io\/ioutil"`
+GREP_IOUTIL=`find --name "*.go" | xargs grep -ns "io\/ioutil"`
```

Comment:
GNU find uses `-name` as the param, `--name` errors with `find: unknown predicate '--name'`. Is `find` in OSX using `--name`?

Reply:
Sorry, it was a mistake which weren't suppose to be committed. Let me revert this file.